app/views/users/registrations/new.html.slim

```diff
@@ -29,5 +32,5 @@
     hr.fr-my-2w
     .rdv-text-align-center
       p
-        | Vous avez un compte ?
+        = "Vous avez un compte ? "
```

Comment:
si tu veux juste ajouter un trailing white space tu peux utiliser `' ` plutôt que `|` 

cf https://rubydoc.info/gems/slim/frames#verbatim-text-with-leading-and-or-trailing-white-space

ou en fait `|>` qui est nettement plus lisible, je ne connaissais pas. et c’est symmétrique avec `=>`

Reply:
Le `|>` ne fonctionne pas dans ce cas, il m’affiche le `>` (ou alors je ne sais pas m’y prendre).
Mais effectivement `=>`/`=<` ça fonctionne
Je vais partir sur un `=<`